resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,73 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Login') }}</div>
```

Comment:
what is the purpose of using `__('Login')` compared to just using normal text `Login`.

Reply:
It's made like this by (php artisan make:auth). Should I modify it?

Reply:
better to change it

Reply:
also others that are using this function

Reply:
Okay, I got it, I'll change it and push.